wiki/Client/Release_stream/Lazer/Feature_comparison/en.md

```diff
@@ -0,0 +1,450 @@
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### Missing a slider head causes a miss
+
+Missing a slider head (either by not hitting it or hitting it during its miss window) would previously break combo but not cause a MISS judgement, and a judgement could still be received for the missed slider by completing the rest of it. This allowed players to receive the S grade while having a low max combo.
+
+In lazer, not hitting the slider head will give a MISS judgement for the whole slider. After missing a slider head, combo, score, and accuracy can still be gained from slider ticks and repeats, while only score and accuracy can still be gained from the slider end.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | No |
+
+#### Slider ends do not cause hitsounds when not hit
+
+In stable, slider ends would play their hitsounds even if they were missed, as long as any part of the slider was hit. This has been changed such that hitsounds match inputs 1:1.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | Yes |
+| Intentionally changed | Yes |
+| Needs further consideration | No |
+
+#### The spinner spin speed cap of 477 RPM has been removed
+
+Instead of a speed cap, spinners now have a score cap. This was done as a simple initial implementation which may be revised in the future.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### Aspire-like glitched sliders are not supported
+
+![](img/aspire-slider.gif)
+
+Some adventurous beatmaps exploit glitches in the stable client that allowed for very weird slider mechanics. These range from zero-length sliders acting as invisible circles, to cross-screen stretched and squished sliders.
+
+More discussion and consideration will be needed for how much of Aspire beatmaps will be compatible going forward. For example, invisible circles might become a properly supported feature in the future.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | No |
+| Needs further consideration | Yes |
+
+### osu!taiko
+
+#### Notes that overlap swells cannot be hit
+
+Some gimmick maps make use of notes that overlap swells.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | No |
+| Needs further consideration | Yes |
+
+#### Drumrolls do not prevent mashing
+
+In stable, drumrolls could not be hit too quickly or too slowly. This restriction has been lifted, just like in ScoreV2.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes? |
```

Comment:
still unsure about this one (https://github.com/ppy/osu-wiki/pull/8791#discussion_r1106846327 didn't really give much of an indication)

Reply:
Dunno, whatever you think.